fe/fe-core/src/main/java/com/starrocks/scheduler/Task.java

```diff
@@ -209,10 +211,22 @@ public void setUserIdentity(UserIdentity userIdentity) {
     }
 
     public String getPostRun() {
-        return postRun;
+        if (source == Constants.TaskSource.MV) {
```

Comment:
it's better to change the behavior at outside rather than changing the simple `getter` function

Reply:
and it's out of the scope of this PR?

Reply:
my mistake, Let me fix it in the next pr...